scripts/plot-comparison.py

```diff
@@ -18,10 +19,11 @@
     "BioBuilds": 118}, orient="index").reset_index()
 pkg_counts.columns = ["source", "count"]
 
+sns.set_palette('colorblind')
```

Comment:
You can set the color palette globally here: https://github.com/bioconda/bioconda-paper/blob/master/scripts/common/__init__.py#L9

Reply:
Good idea, will do.